change library name WiFiNINA -> Adafruit-WiFiNINA

mikeysklar opened the pull request:
The Arduino IDE will re-install the Arduino WiFININA library over the Adafruit-WiFiNINA library on every launch due to the same name being used in library.properties.

Renaming the library to different them would resolve this worked for me.

```
WiFiNINA -> Adafruit-WiFiNINA
```

```
sklarm@grazie libraries % pwd
/Users/sklarm/Documents/Arduino/libraries
sklarm@grazie libraries % head -2 WiFiNINA/library.properties
name=Adafruit-WiFiNINA
version=1.6.0
```

This came up in a [Adafruit Forum issue](https://forums.adafruit.com/viewtopic.php?p=1070994#p1070994) and was easy to replicate.

mikeysklar committed:
change library name WiFiNINA -> Adafruit-WiFiNINA

There might be a more elegant way to do this. The Arduino IDE will re-install the Arduino WiFININA library over the Adafruit-WiFiNINA library on every launch due to the same name being used.

ladyada commented:
use somehting like WifiNINA (Adafruit Fork) since we didnt author this. you also have to open an issue w/arduino library manager to manually update the name

mikeysklar committed:
Renamed library

Following the same convention used for SdFat_-_Adafruit_Fork.

WiFiNINA_-_Adafruit_Fork

mikeysklar commented:
Renamed library. Following the same convention used for SdFat_-_Adafruit_Fork.

WiFiNINA_-_Adafruit_Fork

This WiFiNINA Adafruit Fork has not been included with the Arduino IDE library manager. It would be a new addition.  It has been a [manual zip file download](https://learn.adafruit.com/adafruit-airlift-breakout/arduino#library-install-3024912) on the guide page. 

I can submit a request for the addition. Is it necessary to update the main branch and maybe rev the version to have the right new name in place first?

ladyada commented:
yah just bump the revision number and then you can submit it.

mikeysklar commented:
This WiFiNINA_-_Adafruit_Fork has been added to the Arduino Library Manager.

<img width="350" height="454" alt="Screenshot 2025-11-03 at 6 28 52 PM" src="https://github.com/user-attachments/assets/afd9b564-6097-4bdf-8b34-360e9b30d4c3" />
